docs/storage.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Storage Examples</title>
+    </head>
+    <body>
+        <h1>Storage Examples</h1>
+        <h2>Overview</h2>
+        <p>This page provides examples of different storage configurations.</p>
+
+        <h1>Default Storage</h1>
+        <p>Removing the <code>storageClassName</code> field from the RabbitMQ instance spec will result in each pod attempting to claim a PersistentVolume with the default storage class.</p>
+
+        <p>For this to work seemlessly, you'll need:</p>
```

Comment:
nit: seamlessly

Reply:
Nice catch.  The older I get the worse I am for things like that.